docs/xrunjdwp.md

```diff
@@ -0,0 +1,122 @@
+
+### `launch`
+
+        -Xrunjdwp:launch=<path>
+
+: Starts the debugger process from the specified path. The values of the `address` and `transport` parameters, if available, are also added to the specified path. These parameters help the debugger to connect to the target VM. This option is used with `onthrow`, `onuncaught` or both the parameters to provide *Just-In-Time debugging* in which a debugger process is started when a particular event occurs in this VM.
+
+### `onthrow`
+
+        -Xrunjdwp:onthrow=<exception>
+
+: Delays the debugging process until the specified exception is thrown in this VM. The JDWP agent attaches to the VM and starts debugging when the exception is thrown.
+
+### `onuncaught`
+
+        -Xrunjdwp:onuncaught=[y|n]
+
+: If `onuncaught=y`, delays the debugging process until an uncaught exception is thrown in this VM.
+
+### `server`
+
+        -Xrunjdwp:server=[y|n]
+
+: If `server=n`, the VM does not listen to any incoming debugger connection and it attempts to attach to the local debugger application at the specified [`address`](#address). In this case, specifying the`address` parameter is mandatory. If `server=y`, the target VM acts as a server listening for an incoming debugger connection through the JDWP agent. If `server=y` and `address` is specified, the VM listens for a debugger connection at this address. If `server=y` and `address` is not specified, the VM selects a [`transport`](#transport) address at which to listen for a debugger application.
+
+### `suspend`
+
+        -Xrunjdwp:suspend=[y|n]
+
+: If `suspend=y`, the VM suspends running of the target Java application until the debugger is attached successfully and is ready for debugging. If `suspend=n`, the VM does not suspend running of the target Java application while the debug connection is established. The VM continues running the application as normal. The debugger can attach to the running application at any time.
+
+: When a debug session is restored from a checkpoint, the VM suspends the running of the Java application that was being debugged even if `suspend=n`. The VM waits for the debugger to send a `Resume` request before it restarts the Java application for continuing the debug session. When the debug session is complete, the debugger disconnects from the JDWP agent, and the agent resumes normal running of the Java application.
+
+### ![Start of content that applies to Java 11 (LTS) and later](cr/java11plus.png) `suspendOnRestore`
+
+**(Linux&reg; x86, Linux on POWER&reg; (Little Endian), Linux on AArch64, and Linux on IBM Z&reg; only)**
+
+This parameter is an OpenJ9 specific parameter.
+
+        -Xrunjdwp:suspendOnRestore=[y|n]
+
+: If `suspendOnRestore=y`, the VM suspends the application on CRIU restore and debugging of the application does not happen. If `suspendOnRestore=n`, the VM does not suspend the application on restore, and the debug session is enabled.
```

Comment:
`suspendOnRestore` is similar with `suspend`.
If `suspendOnRestore=y`, the VM suspends the running of the target Java application until the debugger is attached successfully and is ready for debugging upon restore.
If `suspendOnRestore=n`, the VM does not suspend the running of the target Java application while the debug connection is established upon restore.
This is a flag that only affects a VM post-restore.